### PR TITLE
Update Node examples to use new @grpc/proto-loader package

### PR DESCRIPTION
**Pull request by murgatroid99**
Update the Node examples to use the `@grpc/proto-loader` library as we now recommend.

### CODE REVIEW

File: examples/node/dynamic_codegen/route_guide/route_guide_server.js [hunk 1]
```diff
@@ -23,7 +23,16 @@ var parseArgs = require('minimist');
...
+var protoLoader = require('@grpc/proto-loader');
+var packageDefinition = protoLoader.loadSync(
+    PROTO_PATH,
+    {keepCase: true,
+     longs: String,
+     enums: String,
+     defaults: true,
+     oneofs: true
+    });
+var routeguide = grpc.load(packageDefinition).routeguide;
```
Comment (DRoet) [inline examples/node/dynamic_codegen/route_guide/route_guide_server.js]: `var routeguide = grpc.loadPackageDefinition(packageDefinition).routeguide;`

Comment (murgatroid99) [reply examples/node/dynamic_codegen/route_guide/route_guide_server.js]: Thank you

**Comment by murgatroid99** [comment]
Test results are very stale; rerunning.

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 1,976,654      Total (=)      1,976,654

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
10,775,736      Total (>)     10,775,727

 No significant differences in binary sizes


```

**Comment by murgatroid99** [comment]
The basic tests details link says that the tests are passing but the indicator here is not changing.

**Pull request merged**